.github/workflows/scan-bwa.yml

```diff
@@ -7,8 +7,14 @@ on:
   #     - "main"
   #     - "rc"
   #     - "hotfix-rc"
+  # pull_request:
+  #   types: [opened, synchronize, reopened]
+  #   branches-ignore:
+  #     - main
```

Comment:
⛏️ Should this comment have the quotation marks as well so if it gets uncommented it remains equal to the others?